examples2d/add_remove2.rs

```diff
@@ -10,7 +10,7 @@ pub fn init_world(testbed: &mut Testbed) {
     let rad = 0.5;
 
     // Callback that will be executed on the main loop to handle proximities.
-    testbed.add_callback(move |window, physics, _, graphics, _| {
+    testbed.add_callback(move |window, graphics, physics, _, _| {
```

Comment:
This will be needed by the other suggestions that do `&mut window`, and `&mut graphics`:

```suggestion
    testbed.add_callback(move |mut window, mut graphics, physics, _, _| {
```